From 6a1b2ffae0cc12467838bc671e3b089924de90a6 Mon Sep 17 00:00:00 2001
From: Dmitry Babokin <dmitry.y.babokin@intel.com>
Date: Thu, 5 May 2022 16:34:34 -0700
Subject: [PATCH] Bump ISPC version v1.19.0dev

Upstream-Status: Backport

Include only the LLVM specific bits and not the ispc version bump.

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
diff --git a/src/ispc_version.h b/src/ispc_version.h
index 0e28dfcfa..2b9d9b0cc 100644
--- a/src/ispc_version.h
+++ b/src/ispc_version.h
@@ -51,7 +51,7 @@
 #define ISPC_LLVM_15_0 150000
 
 #define OLDEST_SUPPORTED_LLVM ISPC_LLVM_10_0
-#define LATEST_SUPPORTED_LLVM ISPC_LLVM_14_0
+#define LATEST_SUPPORTED_LLVM ISPC_LLVM_15_0
 
 #ifdef __ispc__xstr
 #undef __ispc__xstr
@@ -63,7 +63,7 @@
     __ispc__xstr(LLVM_VERSION_MAJOR) "." __ispc__xstr(LLVM_VERSION_MINOR) "." __ispc__xstr(LLVM_VERSION_PATCH)
 
 #if ISPC_LLVM_VERSION < OLDEST_SUPPORTED_LLVM || ISPC_LLVM_VERSION > LATEST_SUPPORTED_LLVM
-#error "Only LLVM 11.0 - 14.0 are supported"
+#error "Only LLVM 11.0 - 14.0 and 15.0 development branch are supported"
 #endif
 
 #define ISPC_VERSION_STRING                                                                                            \
